radixdlt/src/main/java/com/radixdlt/consensus/LocalSyncRx.java

```diff
@@ -17,22 +17,18 @@
 
 package com.radixdlt.consensus;
 
-import com.radixdlt.crypto.ECPublicKey;
 import com.radixdlt.crypto.Hash;
-import io.reactivex.rxjava3.core.Single;
+import io.reactivex.rxjava3.core.Observable;
 
 /**
- * An asynchronous supplier which retrieves data for a vertex with a given id
+ * Provider of Rx stream of local syncs which are emitted when
+ * the underlying VertexStore has synced to the vertex with the given hash.
  */
-public interface VertexSupplier {
+public interface LocalSyncRx {
 
 	/**
-	 * Execute an RPC to retrieve a vertex given an Id from a node
-	 * TODO: refactor to maintain a unidirectional data flow
-	 *
-	 * @param id the id of the vertex to retrieve
-	 * @param node the node to retrieve the vertex info from
-	 * @return single of a vertex which will complete once retrieved
+	 * Retrieve rx flow of vertex hashes
+	 * @return flow of vertex hashes
 	 */
-	Single<Vertex> getVertex(Hash id, ECPublicKey node);
+	Observable<Hash> localSyncs();
```

Comment:
What makes these 'local'?

Reply:
It's a message originating from a local module